backend/README.md

```diff
@@ -78,6 +78,13 @@ $ npm i
 
 - Run all unit tests and display a report.
 
+Note. Before running any tests:
+
+1. Change the mock flag in config.js to true.
+2. In another terminal, start an instance of the API for the tests to connect to.
+   - This can be done using either `$ npm start` or `$ npm run dev`.
+3. Start the tests
```

Comment:
Perhaps just add " Start the tests by `$ npm start test`.

Reply:
isn't it already 6 lines above? Or do you think it should be repeated?

Reply:
ah right. Yeah sorry i was only looking at the diff and didnt realise its just above. its cool